react/features/base/toolbox/components/web/ToolboxButtonWithIconPopup.tsx

```diff
@@ -33,17 +33,17 @@ interface IProps {
     /**
      * Icon of the button.
      */
-    icon: Function;
+    icon?: Function;
```

Comment:
This component is called ToolboxButtonWithIconPopup, making the icon optional feels wrong.

Reply:
Yep. Will think how to change it. Will probably create a ToolboxButtonWithPopup and reuse it here.